src/game.h

```diff
@@ -443,8 +443,8 @@ class game
 
         void interactive_inv();
 
-        int inv_for_filter( const std::string &title, item_filter filter, const std::string &none_message = "" );
-        int inv_for_filter( const std::string &title, item_location_filter filter, const std::string &none_message = "" );
+        int inv_for_filter( const std::string &title, const item_filter &filter, const std::string &none_message = "" );
```

Comment:
I'm pretty sure the extra code at the call sites is more characters than the wrapper itself.